protos/service_mesh_performance.proto

```diff
@@ -69,6 +69,33 @@ message PerformanceTestResult {
   string test_id = 9;
   string mesh_config_id = 10;
   string env_id = 11;
+  // CPU
+  message CPU {
+    // cpu in mCores
+    int32 cpu_mcores = 1;
+    double user = 2;
+    double nice = 3;
+    double system = 4;
+    double idle = 5;
+    double iowait = 6;
+    double hardware_interrupt = 7;
+    double software_interrupt = 8;
+    double steal = 9;
+  }
+  // Memory
+  message Memory {
+    // memory in MB
+    int32 mem_total_mb = 1;
+    int32 mem_free_mb = 2;
+    int32 mem_used_mb = 3;
+  }
+  // Swap
+  message Swap {
+    // swap in MB
+    int32 swap_total_mb = 1;
+    int32 swap_free_mb = 2;
+    int32 swap_used_mb = 3;
+  }
```

Comment:
Should mem values be `double` as well?

Reply:
@jbyers19 Thanks for reviewing. The mem and swap are calculated in `MB`, it's enough to use `int32` in my opinion.

Reply:
Since these will be used for reporting performance numbers, I believe a higher level of precision would be better. However, I'm new to the project and not familiar with precedents for these types of design decisions so will defer to others on this.

Reply:
Your comments are very helpful and constructive. I set these values as `int32` based on: 

https://github.com/service-mesh-performance/service-mesh-performance/blob/2d986e847ab68a88f9748d0705bbc88aff26cc07/protos/service_mesh_performance.proto#L107

@navendu-pottekkat Do you have any comments on this? Thanks.

Reply:
There doesn't seem to be a reason why we should stick to using `int32`. As @jbyers19 mentioned having a floating point precision makes sense as we want to capture data with higher levels of precision.

@sunkuranganath Do you have thoughts on this?

Reply:
@navendu-pottekkat Make sense, I changed `int32` to `double`.

Reply:
@navendu-pottekkat Can you review it again? Thanks.